frontend/src/lib/components/icons.tsx

```diff
@@ -1399,9 +1398,9 @@ export function IconHeatmap(props: LemonIconProps): JSX.Element {
     )
 }
 
-export function UnverifiedEvent({ width = 24, height = 24, ...props }: React.SVGProps<SVGSVGElement>): JSX.Element {
+export function UnverifiedEvent(props: LemonIconProps): JSX.Element {
     return (
-        <LemonIconBase width={width} height={height} viewBox="0 0 24 24" fill="none" {...props}>
```

Comment:
This change does effect some of the other places it is used in the UI. Makes it more inline with our icon sizing but we need to check how it looks in the Property filters for example

Reply:
(I made the change but wanted to put this in as a reminder)

Reply:
This is actually solved by @Twixes 's changes